MapboxNavigation/RouteVoiceController.swift

```diff
@@ -60,6 +60,12 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     
     override public init() {
         super.init()
+        
+        if !Bundle.main.backgroundModes.audio {
+            print("Voice guidance may not work properly. " +
```

Comment:
Should we throw an error to make this requirement more explicit? If the developer intentionally omitted the `audio` background mode, they can catch the error.